src/til/ut_til/sources

```diff
@@ -0,0 +1,33 @@
+!include ..\..\project.unittest.inc
+
+# -------------------------------------
+# Program Information
+# -------------------------------------
+
+TARGETNAME              = Microsoft.Terminal.Til.UnitTests

```

Comment:
Make sure to add to `/tools/runut.cmd` and the powershell scripts as well

Reply:
got it.